src/components/Modal/VaultStrategyModal/managedVaults/ManagedVaultsStrategyModalApr.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from "react"
+
+import { VaultStrategyModalDefinitionList } from "@/components/Modal/VaultStrategyModal"
+
+export const ManagedVaultsStrategyModalApr: FC = () => {
+  const breakdownApr = {
+    isLoading: false,
+    data: {
+      GMXApr: 0.1,
+      ETHApr: 0.23,
+      totalApr: 0.33,
+    },
+  }
+
+  // we need it as we mock the breakdownApr.data above
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const data = breakdownApr.data as any
+
+  // eslint-disable-next-line no-sparse-arrays
```

Comment:
Can remove this

Reply:
For the demo purpose I can't, without 'any' it generates an error.

Reply:
@vanillaHill this is a different linter rule from the `any` thing. This is for sparse arrays; i.e. arrays without values at certain indices: `[1,,,3,,,,,5]` for example.

Reply:
Right, I was confused